arch/nrf52/nrf52.ini

```diff
@@ -17,7 +17,7 @@ build_flags =
   -DLFS_NO_ASSERT                      ; Disable LFS assertions , see https://github.com/meshtastic/firmware/pull/3818
   -DMESHTASTIC_EXCLUDE_AUDIO=1
   -DMESHTASTIC_EXCLUDE_PAXCOUNTER=1
-  -DMAX_NUM_NODES=80
+  -DMAX_NUM_NODES=70
```

Comment:
Now we found the source of the filesystem badness, do we still need to reduce this?